edge/hoymilesapi.py

```diff
@@ -240,6 +240,8 @@ def get_solar_data(self) -> dict:
     def adjust_solar_data(self, solar_data: dict) -> dict:
         """Adjust solar data like unifed measurements units
 
+        Rename keys: capacitor to array_size and capacitor_kW to array_size_kW
```

Comment:
Ok, so if we want to rename it, maybe it will be better to use name from API response? Or proper form 'capacity'. When I so array_size I start to thinking about some dimensions of my solar installation on roof

Reply:
To be honest, I'm really surprised that this is called in such way. Also google say the same. I think that problem is not between cultures but in translation - main meaning of the word ;)